docs/sql/general/value-expressions.rst

```diff
@@ -48,9 +48,14 @@ identifier. An identifier is an unquoted or double quoted string.
 
 - quoted: ``"columnName"``
 
-It's also possible to include the name of a table or alias in order to
-unambiguously identify a column of a specific relation if a statement contains
-multiple alias or table definitions::
+It's also possible to include the name of a table with or without schema and
+catalog name, or an alias in order to unambiguously identify a column of a
+specific relation if a statement contains multiple aliases or table definitions::
+
+    crate.myschema.mytable.columnname
+    myschema.mytable.columname
+
+or::
```

Comment:
Not sure if necessary, but could add a note section to point out that `crate` is the only available catalog

Reply:
Please double check the `NOTE` change below, thx!